src/main/java/com/moneymatters/repositories/SaleRepository.java

```diff
@@ -11,4 +13,7 @@ public interface SaleRepository extends JpaRepository<Sale, Long> {
     @Query("SELECT COUNT(s) FILTER(WHERE EXTRACT(MONTH FROM s.createdAt) = EXTRACT(MONTH FROM CURRENT DATE )) AS currentMonth, " +
             "(SELECT COUNT(*) FROM Sale s2 WHERE EXTRACT(MONTH FROM s2.createdAt) = (EXTRACT(MONTH FROM CURRENT DATE)-1)) as lastMonth FROM Sale s")
     String getSalesComparedToPastMonth();
+
+    @Query("SELECT s FROM Sale s WHERE EXTRACT(YEAR FROM s.createdAt) = EXTRACT(YEAR FROM CURRENT DATE ) AND EXTRACT(DAY FROM s.createdAt) = EXTRACT(DAY FROM CURRENT DATE ) ORDER BY s.createdAt ASC")
```

Comment:
Is it needed to return a Page?

Reply:
This feature is to return just the sum of the all sales of the day. This information is going to be used by the fist box the home page.

https://money-matters-web.vercel.app/